src/storage/batch_extractor.cc

```diff
@@ -279,15 +281,17 @@ rocksdb::Status WriteBatchExtractor::DeleteCF(uint32_t column_family_id, const S
     std::string user_key;
     std::tie(ns, user_key) = ExtractNamespaceKey<std::string>(key, is_slot_id_encoded_);
 
-    if (slot_id_ >= 0 && static_cast<uint16_t>(slot_id_) != GetSlotIdFromKey(user_key)) {
+    auto key_slot_id = GetSlotIdFromKey(user_key);
+    if (slot_range_.IsValid() && !slot_range_.Contains(key_slot_id)) {
```

Comment:
Is `Contains` enough here?

Reply:
Do you think there are any inappropriate examples? Could you please elaborate?

Reply:
Oh I got this since invalid always return !contains, ignoring this